lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -280,6 +280,25 @@ bool removeExcludeFromRendering(const Ufe::Path& ufePulledPath)
     return true;
 }
 
+//------------------------------------------------------------------------------
+//
+// Turn on the mesh flag to allow topological modifications.
+bool allowTopologyModifications(MDagPath& root)
+{
+    MDGModifier& dgMod = MDGModifierUndoItem::create("Allow topology modifications");
+
+    MItDag dagIt;
+    dagIt.reset(root, MItDag::kDepthFirst, MFn::kMesh);
+    for (; !dagIt.isDone(); dagIt.next()) {
+        MString cmd;
+        if (!cmd.format("setAttr \"^1s.allowTopologyMod\" 1;", dagIt.fullPathName()))
```

Comment:
Surely we can use MDGModifier::newPlugValueBool()?  Starting up the MEL interpreter to set a plug from C++ feels like a bit much.

Reply:
I had looked to MDGModifier, but did not see setAttribute. I'll try your suggestion.